ga/client/rttestimator.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016 Chun-Ying Huang
+ *
+ * This file is part of GamingAnywhere (GA).
+ *
+ * GA is free software; you can redistribute it and/or modify it
+ * under the terms of the 3-clause BSD License as published by the
+ * Free Software Foundation: http://directory.fsf.org/wiki/License:BSD_3Clause
+ *
+ * GA is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * You should have received a copy of the 3-clause BSD License along with GA;
+ * if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef __RTTESTIMATOR_H__
+#define __RTTESTIMATOR_H__
```

Comment:
estimator is a poor name to use here. It's not doing estimation, it's actually getting real measurements.